packages/gatsby-plugin-mdx/gatsby/source-nodes.js

```diff
@@ -221,13 +221,17 @@ ${e}`
         async resolve(mdxNode) {
           const { mdast } = await processMDX({ node: mdxNode })
           const { words } = await getCounts({ mdast })
-          let timeToRead = 0
+          const { timeToRead } = pluginOptions
           const avgWPM = 265
-          timeToRead = Math.round(words / avgWPM)
-          if (timeToRead === 0) {
-            timeToRead = 1
-          }
-          return timeToRead
+          const timeToReadInMinutes = Math.max(
+            1,
+            Math.round(
+              _.isFunction(timeToRead)
+                ? timeToRead(words, mdxNode)
```

Comment:
not match `timeToRead(wordCount, html, md)`

Reply:
👋 I had to add some functionality, but the `timeToRead` signature now matches for both MDX and Remark.